pkg/controller/disruption/disruption_test.go

```diff
@@ -1199,17 +1210,17 @@ func TestUpdatePDBStatusRetries(t *testing.T) {
 	dc, _ := newFakeDisruptionController()
 	// Inject the production code over our fake impl
 	dc.getUpdater = func() updater { return dc.writePdbStatus }
-
+	ctx := context.TODO()
```

Comment:
not sure having this TODO as local variable is better, actually the opposite. Would prefer to keep it every single call